### DISCUSSION TITLE
HTTP access not working properly (only on WiFI AP)

### DISCUSSION THREAD

**Discussion by frankhirsch** [tbnobody/OpenDTU#278] [Q&A]
I did flash the pre-compiled .bin files to my ESP32 board using esptool.py with rebuild binaries from 25th October with no issue, but do not seem to be able to access the stationary website...?

1. Having assigned a static IP in my router, the ESP32 get's the **IP in my local network** 
     - I can ping it
     - port 80 is open (telnet)
     - but my browser does seem to load forever
2. As soon as the WiFI AP (OpenDTU-*) becomes available I can
     - access the website (http://192.168.4.1) and
     - configure my network settings & credentials for the stationary mode

After some tests I did use wget to check the response of the (stationary) website - the result was a compressed (gz) file which is basically the index.html complaining of required JavaScript (which is expected if using wget...). I finally noticed the website does not finalize to load because /js/app.js ist not available in stationary mode.

As I downloaded the rebuild binaries from 25th October I think this should have been fixed in the master?

**Comment by tbnobody** [comment]
I cannot reproduce this issue here.

>  I finally noticed the website does not finalize to load because /js/app.js ist not available in stationary mode.

What response code do you get? What does "not available" mean? Error 404?

**Reply by frankhirsch** [reply]
No, the resource /js/app.js is simply not available... the HTTP request will run (probably forever unless the connection will timeout)
![Bildschirmfoto 2022-10-28 um 18 59 44](https://user-images.githubusercontent.com/1833680/198698174-1a341d46-f039-4451-b90b-891ad92fc02d.png)

**Reply by frankhirsch** [reply]
> I cannot reproduce this issue here.
> 
> > I finally noticed the website does not finalize to load because /js/app.js ist not available in stationary mode.
> 
> What response code do you get? What does "not available" mean? Error 404?

Maybe the following screenshot from the Wifi AP may help as well?
![Bildschirmfoto 2022-10-28 um 19 49 04](https://user-images.githubusercontent.com/1833680/198700684-5f577975-7440-4263-af3a-3f51e036bcc0.png)

**Comment by frankhirsch** [comment]
Any idea what I can do to troubleshoot? 
Trying on different clients/computers and browsers it does sound like a server issue to me - but It‘s quite strange you can not reproduce…

**Comment by frankhirsch** [comment]
Just this moment I tried again - and guess what.. it‘s working on my wife’s iPad. 

I will try again with different devices in different locations and get back in case I was able to reproduce and identify some issues with the mesh network (Linksys Velop). I think I once heard of issues with IOT devices in mesh networks getting in trouble (probably because of some older HTTP protocol…?). So it‘s definitely not an OpenDTU implementation issue.

**Reply by frankhirsch** [reply]
Not sure why, but for some reason it now works on the browsers & computers it didn't before... must have been some weird local network issue.

Thank's anyway for double checking!
